src/rules/select-style.ts

```diff
@@ -51,7 +51,7 @@ export default ESLintUtils.RuleCreator(docsUrl)<Options, MessageIds>({
   defaultOptions: [{ mode: METHOD }],
   create: (context, [{ mode }]) => {
     return {
-      [pipeableSelect](node: TSESTree.CallExpression) {
+      [pipeableSelect(readNgRxStoreNameFromSettings(context.settings))](node) {
```

Comment:
It looks like you removed the type annotation accidentally:

```suggestion
      [pipeableSelect(readNgRxStoreNameFromSettings(context.settings))](node: TSESTree.CallExpression) {
```